test/utils.sh

```diff
@@ -28,48 +56,19 @@ announce() {
 }
 
 has_namespace() {
-  if $cli get namespace  "$1" > /dev/null; then
+  #We dont need retry because some of the scenarios waiting for false
```

Comment:
add a space after #

Reply:
I don't understand the comment. Can you elaborate?

Reply:
When we run retry it will do the same function until it pass or at the end will fail. This specific command handle failure by itself. (If i dont have namespace it will return false and its o.k.) if i will add retry it will not be relevant. clear ?

Reply:
I see. So in this case `false` is not a failure and `wait_for_it` will run for no reason.

Reply:
Please change comment to `We don't need a timeout here as `false` is a valid output. Running with a timeout will run this command repeatedly for no reason, ending with the same result.`

Reply:
right.

Reply:
👍